### PR TITLE
week2 homework

### PR DESCRIPTION
**Pull request by rajanibusani**

### CODE REVIEW

File: week2/materials/homework/assignments.md [hunk 1]
```diff
@@ -4,38 +4,48 @@
...
 
-service name: `?`
+service name: `AWS Certificate Manager` (https://aws.amazon.com/certificate-manager/)
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: `1172 USD (for first 50 TB / Month costs 23usd and after 1TB cost 22usd)`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:19]: Please add come calculations. The values are correct :)

File: week2/materials/homework/assignments.md [hunk 2]
```diff
@@ -4,38 +4,48 @@
...
 
-Total size of website: `?`
+Total size of website: `Total Size: 494.9 KiB`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:26]: 👍🏻

File: week2/materials/homework/assignments.md [hunk 3]
```diff
@@ -4,38 +4,48 @@
...
+Data requests
+Data transfers
+Data retrievals (Glacier only)`
 
 **Assignment 8:**
 
 The file `week2/assignments/products.json` needs to be uploaded Inside the bucket `hyf-serverless-course-week-2`. Upload the file through the CLI and write down the command needed.
 
-docs: 
+docs:aws s3 cp Desktop/hyf-serverless-hm/week2/materials/homework/products.json s3://rajani-week2-homework
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:47]: 👍🏻

File: week2/materials/homework/assignments.md [hunk 4]
```diff
@@ -45,10 +55,17 @@ command:
...
 
-brief description: `?` 
+brief description: `Clean up incomplete multipart uploads.
+Delete previous versions of objects that you don't need.
+Review your storage-class transition costs.
+Review your data retrieval costs.
+Track the requests made to your bucket.
+Check for changes in the size of your bucket.
+Review the cost of individual buckets.
+Understand how your usage relates to your charges`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:70]: 👍🏻

File: week2/materials/homework/assignments.md [hunk 5]
```diff
@@ -4,38 +4,48 @@
...
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/rajani-week2-homework?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `AWS Certificate Manager` (https://aws.amazon.com/certificate-manager/)
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:13]: Please add some explanation :)

**Review by paulinadubas — CHANGES_REQUESTED** [review]

File: week2/materials/homework/assignments.md (hunk 1 above)
Comment (rajanibusani) [reply week2/materials/homework/assignments.md:19]: I added the calculations now...Thank You @paulinadubas

**Pull request merged**